classes/04class/exercises/c04-iac02/carineoliveira/vpc.tf

```diff
@@ -0,0 +1,84 @@
+resource "aws_vpc" "my_vpc" {
+  cidr_block = var.vpc_cidr
+
+  tags = {
+    Name = "devopsacademy-iac"
+  }
+}
+
+resource "aws_subnet" "my_subnets" {
+    count = 4

```

Comment:
instead of specifying the number, you should use this:
count = length(var.subnets)